themes/default/content/docs/intro/vs/cloud-template-transpilers/aws-cdk/_index.md

```diff
@@ -31,7 +31,7 @@ Both Pulumi and CDK allow you to build and deploy infrastructure on AWS using fa
 
 ## Pulumi vs. CDK: Key Differences {#differences}
 
-First, CDK supports only AWS, whereas Pulumi supports over 60 cloud and SaaS providers, with more being added all the time. Second, while both Pulumi and CDK support languages like TypeScript, Python, Go, C#, and Java, the two tools are fundamentally different in how they interpret these languages and deploy resources. Unlike Pulumi, whose open-source engine understands these languages and communicates directly with cloud providers to deploy infrastructure, CDK is a transpiler (i.e., a [source-to-source compiler](https://en.wikipedia.org/wiki/Source-to-source_compiler)) that produces AWS Cloud Assembly, an intermediate format consisting of CloudFormation templates and other files that it uploads to the CloudFormation service, which acts as the deployment engine for provisioning resources.
+First, CDK supports only AWS, whereas Pulumi supports over 60 cloud and SaaS providers, with more being added all the time. Second, while both Pulumi and CDK support languages like TypeScript, Python, Go, C#, and Java, the two tools are fundamentally different in how they interpret these languages and deploy resources. Unlike Pulumi, whose open-source engine understands these languages and communicates directly with cloud providers to deploy infrastructure, CDK is a transpiler (i.e., a [source-to-source compiler](https://en.wikipedia.org/wiki/Source-to-source_compiler)) that produces AWS Cloud Assembly, an intermediate format consisting of CloudFormation templates and assets. After the assets are uploaded to appropriate services (S3, ECR), CDK uses CloudFormation service, which acts as the deployment engine for provisioning resources.
```

Comment:
"Assets" by itself sounds like it refers to something CloudFormation-specific, but my understanding is that in the context of CDK, "assets" essentially means "various other files", so I'd suggest keeping the "other" to eliminate that ambiguity. 

```suggestion
First, CDK supports only AWS, whereas Pulumi supports over 60 cloud and SaaS providers, with more being added all the time. Second, while both Pulumi and CDK support languages like TypeScript, Python, Go, C#, and Java, the two tools are fundamentally different in how they interpret these languages and deploy resources. Unlike Pulumi, whose open-source engine understands these languages and communicates directly with cloud providers to deploy infrastructure, CDK is a transpiler (i.e., a [source-to-source compiler](https://en.wikipedia.org/wiki/Source-to-source_compiler)) that produces AWS Cloud Assembly, an intermediate format consisting of CloudFormation templates and other assets. After the assets are uploaded to the appropriate services (S3, ECR), CDK uses the CloudFormation service, which acts as the deployment engine for provisioning resources.
```